proto/internal/temporal/server/api/history/v1/message.proto

```diff
@@ -48,3 +52,13 @@ message VersionHistories {
     int32 current_version_history_index = 1;
     repeated VersionHistory histories = 2;
 }
+
+message TaskKey {
+    int64 task_id = 1;
+    google.protobuf.Timestamp fire_time = 2 [(gogoproto.stdtime) = true];
+}
+
+message TaskRange {
+    TaskKey inclusive_min_task_key = 1;
+    TaskKey exclusive_max_task_key = 2;
```

Comment:
consider min_task_key_inclusive / max_task_key_exclusive to be consistent with other names used else where.